updatehub/tests/common.rs

```diff
@@ -10,6 +10,12 @@ use std::{env, path::PathBuf};
 pub enum FakeServer {
     NoUpdate,
     HasUpdate(String),
+    CheckRequirementsTest(String),
+}
+
+pub enum StopMessage {
+    Custom(String),
+    Polling(Polling),
```

Comment:
This changes to `FakeServer` and `StopMessage`, and related changes here, doesn't seem to be necessary in this particular commit. If possible it should be moved to the next one where a test with failing checking requirements is actually added